cli/pom.xml

```
@@ -93,9 +98,9 @@
     </dependency>
     <!-- Needed for WireMock test support -->
     <dependency>
-      <groupId>com.github.tomakehurst</groupId>
-      <artifactId>wiremock-jre8</artifactId>
-      <version>2.35.2</version>
+      <groupId>org.wiremock</groupId>
+      <artifactId>wiremock</artifactId>
+      <version>3.11.0</version>
```

very nice that you update here.
With our new module it is more than time that we also go for maven `dependencyMamangement`.
Could you also do that in this PR so we do not introduce dependency versions redundantly in `cli` and `url-updater`?
Simply add a `dependencyMamangement` to our `IDEasy` parent POM and add all dependencies there with their versions. Then remove all these versions from other POMs. If we have multiple artifacts from the same groupId that share the same version, use a variable (like `jackson.version`) in our parent POM.
```suggestion
      <groupId>org.wiremock</groupId>
      <artifactId>wiremock</artifactId>
```

---

Added dependencyManagement to root pom.xml now.